### PR TITLE
Disable the daily upgrade service

### PR DESCRIPTION
**Pull request by viandoxdev**
This should be ok, but I need to disable the other things slowing down the car.

### CODE REVIEW

**Comment by viandoxdev** [comment]
why tho

**Comment by viandoxdev** [comment]
Reopening as it might be useful

File: ansible/donkeycar.yml [hunk 1]
```diff
@@ -18,5 +18,7 @@
       become: no # This role no need root privileges
     - role: ps4-controller # install and configure ds4drv
       become: no # This role no need root privileges
+    - role: optimizations # boot time goes brbrbrbrbr
+      become: yes
```
Comment (yannis-mlgrn) [inline ansible/donkeycar.yml:22]: Better in my-car

File: ansible/roles/optimizations/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Disable daily upgrades
+  systemd:
+    name: apt-daily-upgrade.timer
+    state: disabled
```
Comment (yannis-mlgrn) [inline ansible/roles/optimizations/tasks/main.yml:6]: missing i think some others services, see that : https://superuser.com/questions/1327884/how-to-disable-daily-upgrade-and-clean-on-ubuntu-16-04

**Comment by yannis-mlgrn** [comment]
please fix conflicts @viandoxdev

**Comment by viandoxdev** [comment]
done

**Comment by yannis-mlgrn** [comment]
normally done

File: ansible/roles/optimizations/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- name: Disable daily upgrades
+  systemd:
+    name: "{{ item }}"
+    state: disabled
+    daemon-reload: yes
+  with_items:
+    - 'apt-daily-upgrade.timer'
+    - 'apt-daily.timer'
+    - 'apt-daily.service'
```
Comment (yannis-mlgrn) [inline ansible/roles/optimizations/tasks/main.yml:11]: You need according to the answer of this [problem](https://superuser.com/questions/1327884/how-to-disable-daily-upgrade-and-clean-on-ubuntu-16-04), you need to remove `unattended-upgrades` , stopped `apt-daily.timer` service before disable these services.
The code might be like this : 
```yaml
---

- name: Remove unattended-upgrades package
  ansible.builtin.apt:
    name: unattended-upgrades
    state: absent

- name: stopped daily upgrades
  systemd:
    name: "apt-daily.timer"
    state: stopped

- name: Disable daily upgrades
  systemd:
    name: "{{ item }}"
    state: disabled
    daemon-reload: yes
  with_items:
    - 'apt-daily.timer'
    - 'apt-daily.service'
```

**Review by yannis-mlgrn — CHANGES_REQUESTED** [review]

File: ansible/roles/optimizations/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Disable daily upgrades
+  systemd:
+    name: "{{ item }}"
+    state: disabled
```
Comment (yannis-mlgrn) [inline ansible/roles/optimizations/tasks/main.yml:6]: You have made a mistake here, according to [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html), `state` parameter can't take `disable`. For disable a service you must replace it by `stopped`  and add this line : `enabled: false
### Final code should be : 
```yaml
---

- name: Disable daily upgrades
  systemd:
    name: "{{ item }}"
    state: stopped
    enabled: false
    daemon-reload: yes
  with_items:
    - 'apt-daily-upgrade.timer'
    - 'apt-daily.timer'
    - 'apt-daily.service'
 ```

Comment (yannis-mlgrn) [inline ansible/roles/optimizations/tasks/main.yml:6]: You have made a mistake here, according to [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html), `state` parameter can't take `disable`. For disable a service you must replace it by `stopped`  and add this line : `enabled: false
### Final code should be : 
```yaml
---

- name: Disable daily upgrades
  systemd:
    name: "{{ item }}"
    state: stopped
    enabled: false
    daemon-reload: yes
  with_items:
    - 'apt-daily-upgrade.timer'
    - 'apt-daily.timer'
    - 'apt-daily.service'
 ```

**Review by yannis-mlgrn — CHANGES_REQUESTED** [review]

**Review by yannis-mlgrn — APPROVED** [review]
LGTM

File: ansible/donkeycar.yml [hunk 5]
```diff
@@ -6,7 +6,7 @@
   remote_user: pi
 
   roles:
-    - role: raspi-config #rasperry pi configuration role
+    - role: raspi-config #rasperry pi confiuguration role
```
Comment (Benvii) [inline ansible/donkeycar.yml:9]: Spelling mistake it was good before ^^

**Review by Benvii — APPROVED** [review]
Ok for me

**Pull request merged**